src/model/channel/message.rs

```diff
@@ -72,6 +72,15 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    ///
+    /// **Note**:
+    /// Not all channel mentions in a message will appear in mention_channels.
+    /// Only textual channels that are visible to everyone in a lurkable guild will ever be included.
```

Comment:
I do know this is taken from the official Discord documentation but I personally feel the term *lurkable* is really elusive.
Do you maybe know a term that explains it better? In what way is a guild *lurkable*? I personally do not know what Discord tries to express here.

Reply:
I actually speaking to them about  some issues will ask